quic/s2n-quic-platform/src/io/testing/network.rs

```diff
@@ -76,7 +85,16 @@ impl Buffers {
         }
     }
 
-    pub fn pending_transmissions<F: FnMut(Packet) -> Result<(), ()>>(&self, mut f: F) {
+    pub fn pending_transmission<F: FnMut(&Packet)>(&self, mut f: F) {
+        let lock = self.inner.lock().unwrap();
+        for (_addr, queue) in &lock.tx {
```

Comment:
Fixes the clippy warning

```suggestion
        for queue in lock.tx.values() {
```

Reply:
I think some parts of my vscode installation are broken which is preventing these lints from bubbling up. For now I can just workaround by manually running the same clippy command that we use in CI.